db/seeds/dummy_structures.rb

```diff
@@ -2,122 +2,96 @@
 
 SchoolDataImporter.new(Rails.logger).delay.run
 
-if Cohort.none?
-  Cohort.find_or_create_by!(start_year: 2021)
-  Cohort.find_or_create_by!(start_year: 2022)
+standard_user_emails = ["admin@example.com", "lead-provider@example.com", "second-school-leader@example.com", "school-leader@example.com"]
```

Comment:
I know this won't run in prod but still why do we need it?

Reply:
Because this runs on every review app so that we can log in

Reply:
This is just moved, it was there before